asmcomp/cmmgen.ml

```diff
@@ -3397,11 +3391,7 @@ let curry_function arity =
   else [tuplify_function (-arity)]
 
 
-module IntSet = Set.Make(
-  struct
-    type t = int
-    let compare (x:t) y = compare x y
-  end)
+module IntSet = Numbers.Int.Set
```

Comment:
Here, I would use `module Int = Numbers.Int` again, then this alias can be removed.  Just use `Int.Set` instead of `IntSet`.

Reply:
Done.